modules/jetstream-maxtext-deployment/templates/podmonitoring.yaml.tftpl

```diff
@@ -0,0 +1,15 @@
+apiVersion: monitoring.googleapis.com/v1
+kind: PodMonitoring
+metadata:
+  name: jetstream-podmonitoring
+  namespace: default
+spec:
+  endpoints:
+  - interval: 1s
```

Comment:
This interval seems aggressive for a default - can cost extra $$. Have you considered making this a parameter than customers can configure?